scripts/push_docs_to_branch.sh

```diff
@@ -48,6 +48,9 @@ then
     EXISTING_VERSION=$NEW_VERSION
 fi
 
+# install yarn dependency in the temp directory
+cd $TMP_DOCS_FOLDER/docs && yarn install && cd - || exit 1
```

Comment:
no idea why it didn't fail before, maybe some sort of global cache in `yarn`?! not an expert

Reply:
I have no clue either tbh :sweat_smile: I'm way less aware of the `rasa` side compared to the `rasa-x` side :grimacing:

Reply:
I do think we'll have the same pb for Rasa X 😄